bedrock/externalpages/templates/externalpages/pocket/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends "externalpages/pocket/base.html" %}
```

Comment:
Somehow this file snuck past the license header pre-commit check.

Reply:
Good catch. Removed it. 💫